tethysapp/historical_validation_tool_colombia/templates/historical_validation_tool_colombia/home.html

```diff
@@ -35,29 +35,29 @@
   <div class="hidden" style="margin-right: 15px;">
     {% gizmo text_input geoserver_endpoint %}
   </div>
-  <div class="header-button glyphicon-button" data-toggle="tooltip" data-placement="bottom" title="View Graph">
-    <a data-toggle="modal" data-target="#obsgraph"><span class="glyphicon glyphicon-globe"></span></a>
+  <div class="header-button" data-bs-toggle="tooltip" data-bs-placement="bottom" title="View Graph">
+    <a data-bs-toggle="modal" data-bs-target="#obsgraph"><i class="bi bi-globe"></i></a>
   </div>
 
-  <div class="header-button glyphicon-button" data-toggle="tooltip" data-placement="bottom" title="Help">
-    <a data-toggle="modal" data-target="#help-modal"><span class="glyphicon glyphicon-question-sign"></span></a>
+  <div class="header-button" data-bs-toggle="tooltip" data-bs-placement="bottom" title="Help">
+    <a data-bs-toggle="modal" data-bs-target="#help-modal"><i class="bi bi-question-circle-fill"></i></a>
   </div>
 {% endblock %}
 
 {# Use the after_app_content block for modals #}
 {% block after_app_content %}
-  <!----Observed Data ----->
+  <!-- Observed Data -->
   <div class="modal fade" id="obsgraph" tabindex="-1" role="dialog" aria-labelledby="about-modal-label">
     <div class="modal-dialog" role="document">
       <div class="modal-content">
         <div class="modal-header">
-          <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
           <div class="row">
             <div id="station-info" class="col-md-8"></div>
             <div class="col-md-4">
               <div align="right">
                 <img class="obslogos" src="{% static 'historical_validation_tool_colombia/images/byu_logo.png' %}">
                 <img class="obslogos" src="{% static 'historical_validation_tool_colombia/images/my-icon.png' %}">
+                <button align="right" style="vertical-align: top;" type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"><span aria-hidden="true"></span></button>
```

Comment:
The `<span aria-hidden="true"></span>` is no longer needed. Fix all instances in all files.